tests/public_api/test_connector.py

```diff
@@ -137,7 +138,12 @@ def test_search_model_that_does_not_exist() -> None:
     random_id = Identifier.generate()
     connector = PublicApiConnector.get()
 
-    result = connector.search_model(ExtractedActivity, random_id)
+    try:
+        result = connector.search_model(ExtractedActivity, random_id)
+    except HTTPError as error:
+        if error.response.json().get("message") == "could not create Solr query":
+            pytest.skip("integration test failed due to misconfiguration")
```

Comment:
You're saying that the tests depend on the correct config and fail if the config is wrong. Now if the tests fail/are skipped, it is not directly clear if they are skipped due to wrong config or other problems, right? Is that intended? Or should the correct config be enforced for the tests?

Reply:
to ensure correct field config we'd need to launch the entire "public API" in a github pipeline. the "could not create solr query" message is a strong indicator for a wrong/outdated config, that's why i put that in the if-clause. but you were right, i forgot to re-raise the http error if that message was not in the public-backend's response.

fixed in 218d82a